cmd/api/src/api/v2/saved_queries_test.go

```diff
@@ -42,7 +43,88 @@ import (
 	"go.uber.org/mock/gomock"
 )
 
+// Can only replace a field value at the root of the JSON object
+// if field does not exist in jsonString, will effectively no-op
+func replaceFieldValueInJsonString(jsonString string, field string, value any) (string, error) {
```

Comment:
This function is repeated between both test files in this steel thread. It's likely to show up in most of our API contract tests, so we should spin this off as a useful utility function